app/interfaces/api/entities/search_result.rb

```diff
@@ -109,7 +109,8 @@ def interim_disbursements
       end
 
       def risk_based_bills
-        ((risk_based_class_letter && contains_risk_based_fee).eql?(true) && is_submitted?).to_i
+        ((risk_based_class_letter && (contains_risk_based_fee ||
```

Comment:
I'd refactor to use a couple of methods or more with this method being the parent. just coz all the conditional logic and braces start to hurt peoples heads

Reply:
Thanks Joel, will do.